ssas/credentials.go

```diff
@@ -0,0 +1,9 @@
+package ssas
+
+type Credentials struct {
```

Comment:
I like your suggestion that this might belong better in `systems.go`.

Reply:
done.